scripts/qa_crowbarsetup.sh

```diff
@@ -1472,6 +1472,25 @@ EOF
         -e "s/ [47]00/ $vlan_sdn/g" \
         $netfile
 
+    # set requested networking mode for Crowbar
+    /opt/dell/bin/json-edit -a attributes.network.mode -v "$crowbar_networkingmode" $netfile
+
+    if [ "$crowbar_networkingmode" == "team" ]; then
+        # special conduit mapping when having a crowbar admin node with a single
+        # nic but want bonding for the rest of the nodes
+        local conmap_element='{"pattern": "team/1/crowbar",
```

Comment:
I think the "crowbar" is somewhat redundant. IIRC the role name is used as a pattern and every node will have a role assigned that matches the pattern "crowbar" (think of the node roles, they are called "crowbar_\<nodename\>"). But this is just a minor nitpick.